### ISSUE TITLE
firmware: spurious mqtt packet received

### ISSUE THREAD

**Issue by pat1** [r-map/rmap#134]
I get a not valid packet from mosquitto broker when I use sim800:
```
decimal| binary
-----------------
13     | 0000 1101
10     | 0000 1010
67     | 0100 0011
76     | 0100 1100
79     | 0100 1111
83     | 0101 0011
69     | 0100 0101
68     | 0100 0100
13     | 0000 1101
10     | 0000 1010
```
I can continue with a short MQTT_SOCKET_TIMEOUT

**Comment by pat1** [comment]
solved by #31

**Comment by pat1** [comment]
solved by #31
